WebRoot/public/login.jsp

```diff
@@ -324,6 +324,9 @@
         <%--Forward the user to the initial two factor authentication set up page--%>
         <jsp:forward page="${twoFactorSetupURL}" />
     </c:if>
+    <c:if test="${errorCode eq 'account.WEB_CLIENT_ACCESS_NOT_ALLOWED'}">
+        <zm:logout/>
```

Comment:
this error will come before use is logged into the system, any reason to do a logout here?

Reply:
`zm:login` is called even when ZM_AUTH_TOKEN cookie is sent (authentication has been done). It throws an exception when zimbraWebClientEnabled is FALSE.
If it is changed from TRUE to FALSE after a user is logged in and then the user reloads the page (by pressing F5 key), the exception is thrown. `zm:logout` removes the cookie.